docs/otel-collector-runbook.md

```diff
@@ -0,0 +1,191 @@
+# Deploy otel-collector 
+
+## Deploy an open collector
+
+Deploy an OpenTelemetry Collector [Gateway](https://opentelemetry.io/docs/concepts/data-collection/#deployment) on dokku2.
+
+For interest, here is the [Dockerfile the distribution is built on](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/Dockerfile).
+
+```bash
+dokku apps:create otel-gateway
+# include the -contrib extras inc. httpbasicauthextension
+dokku git:from-image otel-gateway otel/opentelemetry-collector-contrib:0.54.0
+# for the minimal image use:
+# dokku git:from-image otel-gateway otel/opentelemetry-collector:0.54.0
+
+dokku domains:add otel-gateway collector.opensafely.org
+dokku domains:remove otel-gateway otel-gateway.dokku2.ebmdatalab.net
+
+# create a place to store config files
+dokku storage:ensure-directory otel-gateway
+
+dokku config:set otel-gateway HONEYCOMB_KEY=...honeycomb_key...
+```
+
+Contents of `/var/lib/dokku/data/storage/otel-gateway/config.yaml`:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      http:
+
+processors:
+  batch:
+
+exporters:
+  logging:
+    logLevel: info
+
+  otlphttp:
+    endpoint: "https://api.honeycomb.io"
+    headers:
+      "x-honeycomb-team": "${HONEYCOMB_KEY}"
+# required to support metrics spans
```

Comment:
interesting, why is this required for metrics?

Reply:
interesting. I think if this collector was generated the metrics, that's makes, but for metrics generated elsewhere and sent here to be forwarded, we'd want the source collector to set the dataset?